### ISSUE TITLE
Not working on andriod

### ISSUE THREAD

**Issue by yamaan93** [hjmmc/WebGyroForCemuhook#15]
the x, y, x values stay at zero no matter what I do

**Comment by ZainChenDev** [comment]
With version 1.9, open the url and click the blue "this page" button.

**Comment by ArsenicBismuth** [comment]
Yep, using the HTTPS version (port 8443 instead of 8080) as stated in the "this page" solved the problem.
